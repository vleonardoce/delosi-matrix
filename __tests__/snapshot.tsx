/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import Page from "app/page";

it("renders mainpage unchanged", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
