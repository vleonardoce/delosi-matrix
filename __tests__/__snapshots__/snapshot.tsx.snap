// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main>
    <h1
      class="title"
    >
      Matriz
    </h1>
    <form
      class="form"
    >
      <div
        class="mb-2"
      >
        <label
          class="form__label"
          for="dimension"
        >
          Dimensión
        </label>
        <input
          class="form__input "
          id="dimension"
          name="dimension"
          type="number"
        />
      </div>
      <input
        class="form__button"
        type="submit"
        value="Generar"
      />
    </form>
    <section
      class="mb-10"
    />
  </main>
</div>
`;

exports[`renders mainpage unchanged 1`] = `
<div>
  <main>
    <h1
      class="title"
    >
      Matriz
    </h1>
    <form
      class="form"
    >
      <div
        class="mb-2"
      >
        <label
          class="form__label"
          for="dimension"
        >
          Dimensión
        </label>
        <input
          class="form__input "
          id="dimension"
          name="dimension"
          type="number"
        />
      </div>
      <input
        class="form__button"
        type="submit"
        value="Generar"
      />
    </form>
    <section
      class="mb-10"
    />
  </main>
</div>
`;
